chord_metadata_service/chord/workflows/metadata.py

```diff
@@ -147,6 +149,30 @@ def json_file_output(id_: str, output_name: Optional[str] = None):
                 }
             ]
         },
+        WORKFLOW_READDOC: {
```

Comment:
the `readdoc` name doesn't really make sense – it should be `document` I think

Reply:
Sure, it was tentative. Thank you for the insigths!!